tests/smoketests/smoke.js

```diff
@@ -0,0 +1,39 @@
+const tests = [
+  { engine: 'graphviz', file: 'hello.dot' },
+  { engine: 'blockdiag', file: 'kroki.diag' },
+  { engine: 'seqdiag', file: 'sequence.diag' },
+  { engine: 'actdiag', file: 'actions.diag'},
+  { engine: 'nwdiag', file: 'network.diag'},
+  { engine: 'c4plantuml', file: 'banking-system.puml'},
+  { engine: 'ditaa', file: 'components.ditaa'},
+  { engine: 'erd', file: 'schema.erd'},
+  { engine: 'mermaid', file: 'contribute.mmd'},
+  { engine: 'plantuml', file: 'architecture.puml'},
+  { engine: 'svgbob', file: 'cloud.bob'},
+  // { engine: 'nomnoml', file: 'pirate.nomnoml'},
```

Comment:
It's not working ?

Reply:
When I `curl` the nomnoml engine I get this error. :cry: 

```console
$ curl -X POST -H 'Content-type: text/plain' -H 'Accept: svg+xml' --data-binary @tests/smoke/diagrams/pirate.nomnoml http://localhost:80
00/nomnoml/svg
Error 500: pkg/prelude/bootstrap.js:1185
      throw error;
      ^

Error: Cannot find module 'nomnoml'
1) If you want to compile the package/file into executable, please pay attention to compilation warnings and specify a literal in 'require' call. 2) If you don't want to compile the package/f
ile into executable and want to 'require' it from filesystem (likely plugin), specify an absolute path in 'require' call using process.cwd() or process.execPath.
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._resolveFilename (pkg/prelude/bootstrap.js:1287:46)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Module.require (internal/modules/cjs/loader.js:635:17)
    at Module.require (pkg/prelude/bootstrap.js:1166:31)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/snapshot/app/index.js:0:0)
    at Module._compile (pkg/prelude/bootstrap.js:1261:22)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:710:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
 (exit code 1)%
```

And I just noticed thse warnings when I run `make package` for the nomnoml builder container:

```console
$ make package
docker build -f ops/docker/alpine/Dockerfile -t kroki-builder-nomnoml:alpine -t kroki-builder-nomnoml:latest -t kroki-builder-nomnoml:0.6.1-alpine .
Sending build context to Docker daemon  71.68kB
Step 1/9 : FROM node:10.15.0-alpine
 ---> 288d2f688643
Step 2/9 : RUN npm install -g pkg pkg-fetch
 ---> Using cache
 ---> 90cdd61e6d5a
Step 3/9 : ENV NODE node10
 ---> Using cache
 ---> a624dbdc66ca
Step 4/9 : ENV PLATFORM alpine
 ---> Using cache
 ---> 7c1905875633
Step 5/9 : ENV ARCH x64
 ---> Using cache
 ---> e9a231d32196
Step 6/9 : RUN /usr/local/bin/pkg-fetch ${NODE} ${PLATFORM} ${ARCH}
 ---> Using cache
 ---> 791929534cc9
Step 7/9 : COPY . /app
 ---> c011cda81bf6
Step 8/9 : WORKDIR /app
 ---> Running in 0b9d775c3015
Removing intermediate container 0b9d775c3015
 ---> fb232be716f8
Step 9/9 : RUN /usr/local/bin/pkg --targets ${NODE}-${PLATFORM}-${ARCH} . -o app.bin
 ---> Running in 7b1757ee33e6
> pkg@4.4.0
> Warning Cannot find module 'nomnoml' from '/app'
  /app/index.js
> Warning Cannot find module 'nomnoml' from '/app'
  /app/index.js
Removing intermediate container 7b1757ee33e6
 ---> 383287f6031f
Successfully built 383287f6031f
Successfully tagged kroki-builder-nomnoml:alpine
Successfully tagged kroki-builder-nomnoml:latest
Successfully tagged kroki-builder-nomnoml:0.6.1-alpine
```

The pkg error comes from the fact that I didn't run `npm install` in the `nomnoml` directory because then theres is no `node_modules` with `nomnoml` if I don't do that and it will never be in th container as [it's copied into the container](https://github.com/yuzutech/kroki/blob/cce38156565f264bab814d759cf7af306e1fc5e9/nomnoml/ops/docker/alpine/Dockerfile#L9).
But if I run `npm install` before running `make package` in the `nomnoml` directory, the warning disappears but I still get the curl error I told about in the beginning. So it looks like there is a problem with `nomnoml` packaging.

Reply:
> The pkg error comes from the fact that I didn't run npm install in the nomnoml directory because then theres is no node_modules with nomnoml if I don't do that and it will never be in th container as it's copied into the container.

Indeed you need to run `npm install` first on your host.
Which version of Node.js and npm do you use?

> But if I run npm install before running make package in the nomnoml directory, the warning disappears but I still get the curl error I told about in the beginning. So it looks like there is a problem with nomnoml packaging.

I wonder if we can reproduce this issue on Travis? I can't reproduce it on my local machine.